azerothcore/azerothcore-wotlk
# Fix(DB/Locale) Missing borean tundra Locales quests

Borean Tundra has some quests that doesnt have the request_items and offer_reward translated

References taken from: wowhead.com

Valdifer



## Changes Proposed:
-  Update some missing locales that were showing in english

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Built without errors
- Tested in game
- Tested few quests


## How to Test the Changes:


1. .quest start (Choose one of: 11559 ,11560 ,11561 ,11562 ,11563 ,11564 ,11565 ,11566 ,11569 ,11570 ,11571 ,11575 ,11576 ,11582 ,11587 ,11590 ,11599 ,11600 ,11601 ,11603 ,11604 ,11605 ,11607 ,11609 ,11610 ,11612 ,11613 ,11617 ,11620 ,11623 ,11625 ,11626 ,11645 ,11646 ,11648 ,11650 ,11653 ,11658 ,11663 ,11670 ,11671 ,11672 ,11673 ,11679 ,11680 ,11681 ,11682 ,11692 ,11693 ,11694 ,11697 ,11698 ,11699 ,11700 ,11701 ,11704 ,11707 ,11708 ,11710 ,11712 ,11713 ,11715 ,11718 ,11723 ,11725 ,11726 ,11727 ,11728 ,11729 ,11730 ,11733 ,11788 ,11789 ,11790 ,11791 ,11792 ,11793 ,11794 ,11795 ,11796 ,11797 ,11798 ,11864 ,11865 ,11866 ,11867 ,11868 ,11869 ,11870 ,11871 ,11872 ,11873 ,11876 ,11878 ,11879 ,11884 ,11889 ,11892 ,11897 ,11900 ,11901 ,11902 ,11903 ,11904 ,11905 ,11908 ,11910 ,11911 ,11912 ,11913 ,11914 ,11918 ,11919 ,11920 ,11927 ,11928 ,11931 ,11932 ,11936 ,11938 ,11940 ,11941 ,11942 ,11943 ,11944 ,11945 ,11946 ,11949 ,11950 ,11951 ,11956 ,11957 ,11961 ,11962 ,11963 ,11965 ,11967 ,11968 ,11969 ,11973 ,12019 ,12035 ,12086 ,12088 ,12117 ,12141 ,12157 ,12471 ,12517 ,12518 ,12728 ,12791 ,12798 ,13002 ,13004 ,13088 ,13090 ,13094 ,13095 ,13124 ,13128 ,13265 ,13270 ,13311 ,13412 ,13413 ,13414)
2. Talk to NPC to see before complete the text
3. Take a look at objective
4. .quest complete (Take ids from above)
5. Talk to NPC to see after complete the text

## Known Issues and TODO List:

- [Probably i lost some quests due i think when i did the query to show which quests where unfinished in borean tundra seems i forgot to use some parameters or the query was not correct, so please report if you see some quest without translation]
- [ReDo the query]


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
